src/HttpKernel/TestKernel.php

```diff
@@ -34,8 +34,10 @@
      * concrete kernel configure itself using the abstracvt
      * configure() command.
      */
-    public function init()
+    public function __construct($env, $debug)
     {
+        parent::__construct($env, $debug);
```

Comment:
You also have to change the tests, as the mock builder is now ignoring the constructor, meaning the bundle sets won't be registered at all.